src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -289,66 +292,46 @@ export default class PhotoField extends React.Component {
 
-    let fieldView;
-    if (file.uploading) {
-      fieldView = 'progress';
-    } else if (isCropping) {
-      fieldView = 'cropper';
-    } else if (screenReaderError) {
-      fieldView = 'error';
-    } else if (hasFile && !isCropping) {
-      fieldView = 'preview';
-    } else if (!isCropping && !hasFile) {
-      fieldView = 'initial';
-    }
-
-    let uploadMessage;
-    if (smallScreen) {
-      uploadMessage = <span>Upload <i className="fa fa-upload"></i></span>;
-    } else if (fieldView === 'cropper') {
-      uploadMessage = 'Upload a New Photo';
-    } else {
-      uploadMessage = 'Upload Your Photo';
-    }
-
+    let currentLayout = this.state.currentLayout;
+    let uploadButtonText = 'Your Photo';
     let instruction;
-    if (fieldView === 'cropper') {
-      instruction = <span><strong>Step 2 of 2:</strong> Fit your head and shoulders in the frame.</span>;
-    } else if (fieldView === 'initial') {
-      instruction = <span><strong>Step 1 of 2:</strong> Upload a digital photo.</span>;
-    }
-
     let description;
-    if (fieldView === 'cropper') {
-      description = <p>Move and resize your photo, so your head and shoulders fit in the square frame below. Click and drag, or use the arrow and magnifying buttons to help.</p>;
-    } else if (fieldView === 'preview') {
-      description = <div>Success! This photo will be printed on your Veteran ID Card.</div>;
-    } else if (fieldView === 'initial' && this.state.dragAndDropSupported) {
-      description = <p>Drag and drop your image into the square or click the upload button.</p>;
+    let uploadControlClass = classNames('photo-input-container');
+    switch (currentLayout) {
+      case layouts.choosePhoto:
+        description = <p>Drag and drop your image into the square or click the upload button.</p>;
+        instruction = <span><strong>Step 1 of 2:</strong> Upload a digital photo.</span>;
+        break;
+      case layouts.cropPhoto:
+        uploadButtonText = 'a New Photo';
+        instruction = <span><strong>Step 2 of 2:</strong> Fit your head and shoulders in the frame.</span>;
+        description = <p>Move and resize your photo, so your head and shoulders fit in the square frame below. Click and drag, or use the arrow and magnifying buttons to help.</p>;
+        break;
+      case layouts.watchUpload:
+        break;
+      case layouts.previewPhoto:
+        description = <div>Success! This photo will be printed on your Veteran ID Card.</div>;
+        uploadControlClass = classNames('photo-input-container', 'photo-input-container-left');
+        break;
+      case layouts.screenReaderError:
+        uploadControlClass = classNames('photo-input-container', 'photo-input-container-left');
+        break;
+      default:
+        currentLayout = layouts.choosePhoto;
+        description = <p>Drag and drop your image into the square or click the upload button.</p>;
```

Comment:
This switch statement consolidates several instance variables and if statements.